data/avgi/avgi ships.txt

```diff
@@ -0,0 +1,2142 @@
+	bay "Drone" -63 -219
+	bay "Drone" 63 -219
+	bay "Drone" -18 -216
+	bay "Drone" 18 -216
+	bay "Drone" -24 -175
+	bay "Drone" 24 -175
+	bay "Drone" -24 -175
+	bay "Drone" 24 -175
+	bay "Drone" -46 -165
+	bay "Drone" 46 -165
+	bay "Drone" -46 -165
+	bay "Drone" 46 -165
+	bay "Drone" -18 -150
+	bay "Drone" 18 -150
+	bay "Fighter" 0 -216
+		over
+	bay "Fighter" 0 -216
+		under
+	bay "Fighter" 0 -150
+		over
+	bay "Fighter" 0 -150
+		under
+	bay "Fighter" 0 -117
+		over
+	bay "Fighter" 0 -117
+		under
+
+	leak "leak" 30 60
+	leak "flame" 30 90
+	leak "big leak" 30 30
+	explode "tiny explosion" 36
+	explode "small explosion" 42
+	explode "medium explosion" 45
+	explode "large explosion" 36
+	explode "huge explosion" 6
+	"final explode" "final explosion large"
+
+	description "The Atenuatia is an Avgi fleet carrier, a capital ship that makes up the centerpiece of their battlefleets. With six wings of combat drones and two wings of bombers, an Atenuatia can bring forth an astounding amount of firepower. Carrying this arsenal of fighters has its cost, however, and the Atenuatia can carry relayively few heavy weapons for its size."
+
+
+# Fighters
+
+ship "Akoustiki"
+	sprite "ship/avgi akoustiki"
+	thumbnail "thumbnail/avgi akoustiki"
+	attributes
+		category "Fighter"
+		licenses
+			"Avgi Civilian"
+		"shields" 180
+		"hull" 1200
+		"required crew" 1
+		"bunks" 2
+		"mass" 36
+		"drag" 0.9
+		"heat dissipation" 0.9
+		"fuel capacity" 100
+		"cargo space" 30
+		"outfit space" 96
+		"weapon capacity" 24
+		"engine capacity" 24
+		weapon
+			"blast radius" 12
+			"shield damage" 240
+			"hull damage" 180
+			"hit force" 120
+	outfits
+		"Speckle Railgun"
+		"Speck Round" 1800
+		
+		"Concentrating Solar Array"
+		"Ultracapacitor Cell" 3
+		"Multispectral Scanner"
+		"Spectrometer Array"
+		"Electromotive Rifle" 2
+		
+		"Magnetoplasma Drive"
+
+	gun -18 -15 "Speckle Railgun"
+	gun 18 -15
+
+	engine -12 18
+	engine 12 18
+	"reverse engine" -9 -15
+		angle -30
+		zoom 0.5
+	"reverse engine" 9 -15
+		angle 30
+		zoom 0.5
+	"steering engine" -9 -15
+		angle 120
+		zoom 0.5
+		right
+	"steering engine" 9 -15
+		angle -120
+		zoom 0.5
+		left
+	"steering engine" -15 12
+		angle 60
+		zoom 0.5
+		left
+	"steering engine" 15 12
+		angle -60
+		zoom 0.5
+		right
+
+	leak "leak" 30 60
+	explode "tiny explosion" 9
+
+	description "The Akoustiki Excursion Vehicle is a modern take on the utility pod, a small personal craft designed to aid in the maintenance and repair of a station, or a carrier vessel and its escorts. It is just as commonly seen flitting about the orbitals, servicing satellites acting as a personal transport shuttle between stations and low gravity bodies. Recent advances in magnetoplasmadynamic engines have made it a capable mining craft as well, albeit one with little room for ores."
+
+
+ship "Photikon"
+	sprite "ship/avgi photikon"
+	thumbnail "thumbnail/avgi photikon"
+	attributes
+		category "Fighter"
+		licenses
+			"Avgi Military"
```

Comment:
```suggestion
		licenses
			"Avgi Military"
		"cost" 138000
```
Similar to the Dagger (129,000), but slightly higher due to the turret mount and greater HP total.